vhost-device-console/src/main.rs

```diff
@@ -54,11 +54,21 @@ struct ConsoleArgs {
     #[clap(short = 's', long, value_name = "SOCKET")]
     socket_path: PathBuf,
 
+    /// Virtual machine communication endpoint.
+    /// Unix domain socket path (e.g., "/tmp/vm.sock").
+    #[clap(
+        short = 'v',
+        long,
+        value_name = "VM_SOCKET",
+        default_value = "/tmp/vm.sock"
+    )]
+    uds_path: PathBuf,
```

Comment:
Should we prevent that `tcp_port` and `uds_path` are specified together?

Reply:
#[clap(short = 'p', long, value_name = "PORT", default_value = "12345")]
We will always get 12345 since tcp-port has a default_value, even if the mode isn't 'network'.
Should we explicitly make these parameters mutually exclusive and error out for users, or silently ignore unused parameters? (I lean toward making them mutually exclusive and explicitly erroring out.)

Reply:
IMHO we should make these parameters mutually exclusive. We can also remove the default port value, not sure why we chosed that one.